.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Details
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide any publicly-available historical context._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any publicly-available hardware peculiarities._
+
+**Additional Documentation:**  
```

Comment:
Can we just make the title to "** Link to github issues or internal JIRA **"? So there is no confusion what should be "documentation" or "additional information".

Reply:
** Link to GitHub issue# (if applicable)** is fine, do not mention JIRA.

Reply:
I've addressed this elsewhere without mention of JIRA.